lib/compiler-finder.js

```diff
@@ -147,6 +147,8 @@ class CompilerFinder {
             group: group,
             groupName: props("groupName", ""),
             includeFlag: props("includeFlag", "-isystem"),
+            includePath: props("includePath", ""),
```

Comment:
I agree with Matt in that this could be done elsewhere. I think I have a good solution in mind, give me a bit of time

Reply:
I don't understand this; `includePath` is where your includes live; you should point these at your standard and system libraries. It'll get picked up for any compiler on Windows. And if you use, say, vcpkg, you can point it at that include directory as well.

Reply:
Oh, that makes sense, I did not fully understand its pourpose. Thanks for the explanation :D